src/getUTCOffsetFromDate.ts

```diff
@@ -0,0 +1,10 @@
+import dayjs from 'dayjs';
+import {isString} from 'lodash';
```

Comment:
This package do not depend on lodash. 
Please use typeof === 'string' alternative

Reply:
thx, didn't notice